tests/test_printer.py

```diff
@@ -338,6 +338,17 @@ def test_print_custom_name():
     assert_print_op(module, expected, None)
 
 
+def test_print_custom_block_arg_name():
+    block = Block.from_arg_types([i32, i32])
+    block.args[0].name = "test"
+    block.args[1].name = "test"
+
+    io = StringIO()
+    p = Printer(target=Printer.Target.MLIR, stream=io)
+    p.print_block(block)
+    assert io.getvalue() == """\n^0(%test : i32, %0 : i32):"""
```

Comment:
Why do we have a `\n` here at the start already? Does print_block add that?

Reply:
Yep!